apps/backend/src/modules/users/user.repository.ts

```diff
@@ -59,8 +68,17 @@ class UserRepository implements Repository {
 		};
 	}
 
-	public async findByEmail(email: string): Promise<null | UserEntity> {
-		const user = await this.userModel.query().findOne({ email });
+	public async findByEmail(
+		email: string,
+		includeDeleted = false,
```

Comment:
Let's use more common prefix for boolean variables
```suggestion
		hasDeleted = false,
```